### DISCUSSION TITLE
How do I get a username that’s taken?

### DISCUSSION THREAD

**Discussion by zira** [community/community#23934] [New to GitHub]
<p>What the title says……</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>GitHub usernames are offered on a first-come first-served basis. You can always contact the owner of the username if they have contact information available, but usually the answer is that you’d have to choose a different username.</p>

**Comment by zira** [comment]
<p>Thank you for your help</p>
